index.bs

```diff
@@ -340,8 +340,17 @@ Issue: Include cross references to the specs for these hash functions.
     value.  This signals to the listening agent that it should connect to the
     advertising agent to discover updated metadata.
 
-Issue: Add examples of sample mDNS records.
+The advertising agent should add an additional field to the TXT record:
+
+: pw
+:: An alphanumeric, unguessable token consisting of characters from the set
+    `[A-Za-z0-9+/]`.
```

Comment:
Why allow forward slash '/' and plus sign '+'?

Reply:
This is the same character set as Base64.  It allows agents to use Base64 (which is widely implemented) to encode a binary token.  But per earlier feedback we're not going to require Base64 in case agents want to make a different choice.